dogowner/test_dog_owner.py

```diff
@@ -43,140 +13,153 @@ def test_create_dog_owner(self, create_dog_owner_user):
         assert create_dog_owner_user in DogOwner.objects.all()
 
     def test_del_dogowner(self, create_dog_owner_user):
-        dogOwner_01 = DogOwner.objects.get(first_name=FIRST_NAME)
+        dogOwner_01 = DogOwner.objects.get(first_name='NEW')
         dogOwner_01.user.delete()
         assert dogOwner_01 not in DogOwner.objects.all()
         assert dogOwner_01.user not in User.objects.all()
 
     def test_duplicate_creation_dog_owner_user(self):
         with pytest.raises(ValidationError, match="Invalid username - username already exist"):
             for i in range(2):
-                DogOwner.create(email=EMAIL, username='SAME_USER_NAME_03', password=PASSWORD,
-                                dog_name=DOG_NAME, first_name=FIRST_NAME, last_name=LAST_NAME,
-                                phone_number=PHONE_NUMBER, dog_race=DOG_RACE, dog_picture_url=DOG_PICTURE_URL,
-                                dog_age=DOG_AGE, dog_weight=DOG_WEIGHT, dog_gender=DOG_GENDER
+                DogOwner.create(email='testuser@gmail.com', username='SAME_USER_NAME_03', password='testpassowrd',
```

Comment:
hard coded values prone to bugs

Reply:
How so? in static data, you might do better with explicit data.  if we're afraid of bugs, we can test it.